### DISCUSSION TITLE
Allow disabling deep linking on iPhone App

### DISCUSSION THREAD

**Discussion by ericoporto** [community/community#69388] [Mobile]
### Select Topic Area

Product Feedback

### Body

I would prefer to have the mobile app as a plus and still open links to repositories in Safari. I really don't want to open repositories in the web app - the navigation is just not nice and any sort of code editing to make a PR or something through the app is really not supported there. So just give some way to disable this.

I still want the mobile app - mostly for the authentication feature and a little for the notifications there too.

**Comment by salimovrashad** [comment]
Response #1
